collections/open-source-organizations/index.md

```diff
@@ -14,6 +14,8 @@ items:
  - IBM/ibm.github.io
  - Microsoft/microsoft.github.io
  - artsy/artsy.github.io
+ - Google/google.github.io
+ - Facebook/facebook.github.io
```

Comment:
This repo doesn't appear to exist: https://github.com/Facebook/facebook.github.io